src/components/verification/index.tsx

```diff
@@ -0,0 +1,192 @@
+        boat={boatInfo}
+        booking={data as unknown as Booking}
+        onSuccess={() => router.replace("/success")}
+        bookingId={bookingId}
+        steps={["validateFront", "validateBack", "uploadFrontIdImage", "uploadBackIdImage", "pay", "saveData", "notifyCustomer"]}
+      />
+      <div className="flex md:w-[77%] w-full  justify-center items-center md:p-6 p-2">
+        <div className="bg-white md:w-[70%] w-full rounded-lg">
+          <p className="text-black flex items-center justify-center mt-4 font-semibold md:text-xl text-sm">
+            {t("title.verification_form")}
+          </p>
+          <form onSubmit={formik.handleSubmit}>
+            <div className="md:p-6 sm:p-8 p-6">
+              <div className="flex justify-between w-full">
+                <FormWrapper>
+                  <CommonLabel input="text">
+                    {t("input.first_name")}
+                  </CommonLabel>
+                  <CommonInput
+                    type="text"
+                    name="FirstName"
+                    id="firstname"
+                    placeholder={t("input.first_name")}
+                    value={data["First Name"]}
+                    onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                      setData({ ...data, "First Name": e.target.value })
+                    }
+                    required={true}
+                  />
+                </FormWrapper>
+                <FormWrapper>
+                  <CommonLabel input="text">
+                    {t("input.last_name")}
+                  </CommonLabel>
+                  <CommonInput
+                    type="text"
+                    name="Last Name"
+                    id="lastname"
+                    placeholder={t("input.last_name")}
+                    value={data["Last Name"]}
+                    onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                      setData({ ...data, "Last Name": e.target.value })
+                    }
+                    required={true}
+                  />
+                </FormWrapper>
+              </div>
+              <div className="mt-6 flex items-baseline">
+                <div className="flex items-center mb-4">
+                  <input
+                    id="default-radio-1"
+                    type="radio"
+                    value="nationalId"
+                    checked={data.documentType === "National ID"}
+                    name="document-radio"
+                    onChange={() =>
+                      setData({ ...data, documentType: "National ID" })
+                    }
+                    required
+                    className="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 dark:ring-offset-gray-800 dark:bg-gray-700 dark:border-gray-600"
+                  />
+                  <label className="ms-2 md:text-base text-sm  text-black">
+                    {t("input.national_id")}
+                  </label>
+                </div>
```

Comment:
This input shall be common component, there is at least 6 on the code I think so far